src/pages/Admin/VenueEventDetails.tsx

```diff
@@ -49,6 +50,34 @@ const VenueEventDetails = ({
           <RenderMarkdown text={description} key={`${description}#${index}`} />
         ))}
       </div>
+      <div className="button-container">
+        {isEditable && (
+          <div>
+            <div>
+              <button
+                role="link"
+                className="btn btn-primary"
+                onClick={() => {
+                  setEditedEvent && setEditedEvent(venueEvent);
+                  setShowCreateEventModal(true);
+                }}
+              >
+                Edit
+              </button>
```

Comment:
Where was this taken from?

Reply:
It's the way it was done before edit/delete was deleted